packages/server/graphql/mutations/deleteComment.ts

```diff
@@ -45,8 +46,8 @@ const deleteComment = {
       return {error: {message: `Not a member of the meeting`}}
     }
     const {createdBy} = comment
-    if (createdBy !== viewerId) {
-      return {error: {message: 'Can only delete your own comment'}}
+    if (createdBy !== viewerId && createdBy !== PARABOL_AI_USER_ID) {
```

Comment:
We need to check if the comment belongs to the meeting, otherwise I can delete any AI comment in database